msteams-platform/sbs-botbuilder-taskmodule.yml

```diff
@@ -85,7 +85,7 @@ items:
        You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
     
     1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
-
+       
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
There is no extra space which needs to be removed, there is an image after the text, hence the space, which I dont think should be removed. Please advise.